aiidalab_launch/instance.py

```diff
@@ -158,7 +158,7 @@ def pull(self) -> docker.models.images.Image | None:
             self._image = image
             return image
         except docker.errors.NotFound:
-            LOGGER.warn(f"Unable to pull image: {self.profile.image}")
+            LOGGER.warning(f"Unable to pull image: {self.profile.image}")
```

Comment:
`logging.warn` is deprecated, and we were getting a warning from pytest about this.